src/.vuepress/components/common/codepen-snippet.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <p
+    class="codepen"
+    data-theme-id="39028"
+    data-preview="true"
+    data-editable="true"
+    :data-height="height"
+    :data-default-tab="tab"
+    :data-user="user"
+    :data-slug-hash="slug"
+    :data-pen-title="title"
+    :style="`height: ${height}px`">
+    <span>See the Pen <a :href="href">{{ title }}</a>
+    by {{ name ? name : user }} (<a :href="`https://codepen.io/${user}`">@{{user}}</a>)
+    on <a href="https://codepen.io">CodePen</a>.</span>
+  </p>
+</template>
+
+<script>
+export default {
+  props: {
+    team: {
+      type: Boolean,
+      default: true,
+    },
+
+    user: {
+      type: String,
+      default: 'Vue',
+    },
+
+    name: {
+      type: String,
+      default: null,
+    },
+
+    title: {
```

Comment:
**nitpick**: should we make `title` and `slug` required? This way it would be easier to see that we didn't miss important ones.